From 59b8db5f8f401a6c45854c676e603916d5bab10a Mon Sep 17 00:00:00 2001
From: vertigo235 <jbjonesclt@gmail.com>
Date: Sat, 27 Apr 2019 10:04:48 -0400
Subject: [PATCH] Use 4mm instead of 2mm on the "Load More" check

---
 Firmware/mmu.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Firmware/mmu.cpp b/Firmware/mmu.cpp
index ef5b021c..409f4baa 100644
--- a/Firmware/mmu.cpp
+++ b/Firmware/mmu.cpp
@@ -1461,7 +1461,7 @@ static void load_more()
     }
     current_position[E_AXIS] += 60;
     plan_buffer_line(current_position[X_AXIS], current_position[Y_AXIS], current_position[Z_AXIS], current_position[E_AXIS], MMU_LOAD_FEEDRATE, active_extruder);
-    current_position[E_AXIS] -= 58;
+    current_position[E_AXIS] -= 56;
     plan_buffer_line(current_position[X_AXIS], current_position[Y_AXIS], current_position[Z_AXIS], current_position[E_AXIS], MMU_LOAD_FEEDRATE, active_extruder);
     st_synchronize();
 }
-- 
2.15.0.windows.1

